src/components/Pages/ActionsPage/ActionCard.js

```diff
@@ -266,6 +270,22 @@ class ActionCard extends React.Component {
                 >
                   Info
                 </MEButton>
+                {!action?.is_published && (
```

Comment:
I think this may be a problem for the Sandbox.  Make sure Edit button only shown for the user who submitted the action.  Maybe don't show this button if Sandbox mode.

Reply:
the user submitted content are not queried in sandbox mode

Reply:
and yes only the creator of the content can see and edit for as long as it is not live